### DISCUSSION TITLE
Unreachable code in livewire.esm.js

### DISCUSSION THREAD

**Discussion by sohailahmad07** [livewire/livewire#6065] [7. Bugs (archived)] [answered]
### Livewire version

3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

HI 
there is unreachable code it livewire.esm.js I don't know if I intentional or a bug I just get a warning in the console 
unreachable statement here the code at line 6757

```
function transitionOut(body) {
      return;
      body.style.transition = "all .5s ease";
      body.style.opacity = "0";
    }
    function transitionIn(body) {
      return;
      body.style.opacity = "0";
      body.style.transition = "all .5s ease";
      requestAnimationFrame(() => {
        body.style.opacity = "1";
      });
    }
```

**Comment by joshhanley** [comment]
@sohailahmad07 thanks for reporting! It looks like a bug. Can you share a screenshot of the warning? Thanks!

**Comment by olehlong** [comment]
Same issue
![image](https://github.com/livewire/livewire/assets/3706152/c86ed27c-f353-430d-8535-389147dcb54a)
![image](https://github.com/livewire/livewire/assets/3706152/0fc564d1-b696-4747-be6c-472ff3c28639)

**Comment by joshhanley** [comment, marked as answer]
@sohailahmad07 and @olehlong thanks! It looks like Caleb added the early return to disable the function for now https://github.com/alpinejs/alpine/commit/d2943ddfdd496d53576c9bf03b66677bfa84e431#diff-d60b88fa5002f1b47304b162bb2c46d007502d7c34138ce5f72b2be53a79569cR28
